pkg/network/blockqueue.go

```diff
@@ -15,6 +15,10 @@ type blockQueue struct {
 	relayF      func(*block.Block)
 }
 
+// blockCacheSize is the amount of blocks above current height
+// which are stored in queue.
+const blockCacheSize = 10000
```

Comment:
`2000`? Consider 256K blocks and the fact that 2000 blocks per second is a natural verification limit. That's also four times more than the standard block request would return (500) which is a nice number wrt #1376.

Reply:
Fixed.